package.json

```diff
@@ -26,7 +26,7 @@
     "axios": "^0.20.0",
     "dotenv": "^8.2.0",
     "find-process": "^1.4.3",
-    "forever-monitor": "3.0.1",
+    "forever-monitor": "3.0.2",
```

Comment:
Why not ^3.0.2?   I know the original specified exact, but if you don't see a problem now, let's switch this over to ^